cmd/ctr/commands/run/run.go

```diff
@@ -90,40 +90,40 @@ func parseMountFlag(m string) (specs.Mount, error) {
 
 // Command runs a container
 var Command = cli.Command{
-	Name:           "run",
-	Usage:          "run a container",
-	ArgsUsage:      "[flags] Image|RootFS ID [COMMAND] [ARG...]",
-	SkipArgReorder: true,
```

Comment:
Is there an equivalent  for v2?

Reply:
It seems the flag reordering is removed from v2.

https://github.com/urfave/cli/blob/45952a7d1b0e956db2099493fe1a42837ff5bd0d/docs/CHANGELOG.md#1190---2016-11-19

> SkipArgReorder was added to allow users to skip the argument reordering. This is useful if you want to consider all "flags" after an argument as arguments rather than flags (the default behavior of the stdlib flag library). This is backported functionality from the removal of the flag reordering in the unreleased version 2